labs/lab6/lab6.adoc

```diff
@@ -29,56 +37,311 @@ Password: INSTRUCTOR WILL PROVIDE
 
-link:https://tower-<student_id>.labs.sysdeseng.com[https://tower-<student_id>.labs.sysdeseng.com] 
+Build the apb which will execute a Docker build using the _Dockerfile_ located within the _alertmanager-tower-bridge-apb_ directory.
+
+.master$
+[source, bash]
+----
+apb build
+----
+
+A new image containing the APB was created locally. This can be confirmed by listing all of the images on the machine and locating the image called _alertmanager-tower-bridge-apb_.
+
+.master$
+[source, bash]
+----
+docker images
+----
+
+The final step in the Ansible Playbook Bundle creation process is to push the image from the local machine to OpenShift's internal registry. In order to communicate with the registry, an authenticated user to the platform must be used as they contain an OAuth token needed to facilitate the communication.
+
+Login as the "<student_id>-admin" user which has elevated cluster privileges:
+
+.master$
+[source, bash]
+----
+oc login -u <student_id>-admin
+----
+
+Push the APB to the OpenShift integrated registry
+
+.master$
+[source, bash]
+----
+apb push
+----
+
+Confirm the APB is available in the Ansible Service broker by listing all registered APB's.
+
+.master$
+[source, bash]
+----
+apb list
+----
+
+Since no additional interaction is needed with OpenShift's registry at this time, login to OpenShift using the _system:admin_ user.
+
+.master$
+[source, bash]
+----
+oc login -u system:admin
+----
+
+== Deploying the AlertManager-Tower-Bridge Application
+
+The AlertManager-Tower-Bridge application aid in providing cluster capabilities to dynamically scale the size of the OpenShift. Since the application is loosely tied to the Prometheus ecosystem and specifically AlertManager, it should be deployed alongside Prometheus in the `openshift-metrics` project. Given this project requires elevated rights, the _<student_id>-admin_ should be utilized. 
+
+Navigate to the OpenShift Web Console and login as the _<student_id>-admin_ user.
+
+link:https://master-<student_id>.labs.sysdeseng.com:8443[https://master-<student_id>.labs.sysdeseng.com:8443]
+
+After logging in, locate "My Projects" on the righthand side and select **View All** to view all projects the user has access to. Select **openshift-metrics**.
+
+To deploy the APB to the project, select **Add to Project** and then **Browse Catalog** from the link on the top righthand side of the page.
```

Comment:
"To deploy the APB to the project, in the right hand side of the top navigation panel, select...."